anycable.gemspec

```diff
@@ -15,13 +15,14 @@ Gem::Specification.new do |spec|
   spec.homepage      = "http://github.com/anycable/anycable"
   spec.license       = "MIT"
 
-  spec.executables   = ["anycable"]
+  spec.executables   = %w[anycable anycabled]
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.require_paths = ["lib"]
 
   spec.required_ruby_version = '>= 2.4.0'
 
   spec.add_dependency "anyway_config", "~> 1.4.2"
+  spec.add_dependency "daemons", "~> 1.3"
```

Comment:
> we can make it optional and ask users to require it on their own

Let's do this)

We can add a user-friendly message to `anycabled` with the instructions on how to install the dependency.

See, for example:
- https://github.com/palkan/test-prof/blob/master/lib/test_prof/ruby_prof.rb#L204-L223
- https://github.com/palkan/test-prof/blob/master/lib/test_prof.rb#L50-L56
- https://github.com/palkan/test-prof/blob/master/lib/test_prof/utils.rb#L7-L16

Reply:
Done